.gemini.yml

```diff
@@ -9,7 +9,19 @@ system:
 browsers:
   chrome:
     desiredCapabilities:
-      browserName: chrome
-      version: 54.0
-      platform: MacOs 10.12
-      name: gemini visual regression tests
+      browserName: 'chrome'
+      version: '54.0'
+      platform: 'macOs 10.12'
+      name: 'gemini visual regression tests chrome'
```

Comment:
the browser name isn't needed since SauceLabs already reports that in their dashboard.